templates/jenkins.service.j2

```diff
@@ -7,10 +7,8 @@ Environment="CASC_JENKINS_CONFIG={{ jenkins_config_dir }}/jenkins.yaml"
 Environment="JENKINS_HOME={{ jenkins_home }}"
 ExecStart={{ jenkins_command }}
 # So systemd won't generate a PID file for non-forking services, so we need to make one.
-# Here we grep only for the WAR filename, not the whole process name, or else quotes and
```

Comment:
> While pgrep guarantees that its root process won't be matched, this does
not apply to the root process of that process,

Why are you talking about "root process"? That seems out of topic.

Reply:
I've reworded the commit message to be a bit clearer. See c107143 (and sorry for the force-push).